app/experimenter/experiments/models/nimbus.py

```diff
@@ -323,7 +323,12 @@ def proposed_end_date(self):
     def computed_enrollment_days(self):
         paused_change = (
             self.changes.all()
-            .filter(experiment_data__is_paused=True)
+            .filter(
+                experiment_data__is_paused=True,
+                new_status=NimbusExperiment.Status.LIVE,
+                new_status_next=None,
+                new_publish_status=NimbusExperiment.PublishStatus.IDLE,
+            )
```

Comment:
Ah yep true, another great catch from @heres-maria.  You might want to move these filters into a Q object on the model with a label just so it's declared clearly if you're so inclined, but this looks right to me.